src/openforms/forms/models/form_statistics.py

```diff
@@ -0,0 +1,43 @@
+from django.db import models
+from django.utils.formats import localize
+from django.utils.translation import gettext_lazy as _
+
+
+class FormStatistics(models.Model):
+    form = models.ForeignKey(
+        verbose_name=_("form"),
+        to="forms.Form",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+    )
```

Comment:
shouldn't this be `OneToOneField` instead of FK? Having multiple statistics for the same form is going to cause problems, no?